itests/testdata/reward_distribution_features_testdata.go

```diff
@@ -0,0 +1,465 @@
+			XtnDiffBalance:       MaxAddressReward,
+			Term:                 utl.GetRewardTermAfter20Cfg(suite),
+		})
+}
+
+// 2 miners, dao, xtn, initR=500000000, increment = 100000000, desiredR = 300000000
+// ("preactivated_14_19_20/5W_2miners_dao_xtn_decrease.json")
+// ("preactivated_14_supported_19_20/5W_2miners_dao_xtn_decrease.json")
+// NODE - 816
+func GetRewardDecreaseDaoXtnTestData(suite *f.BaseSuite, addresses AddressesForDistribution,
+	height uint64) RewardDistributionTestData[RewardDistributionExpectedValues] {
+	return NewRewardDistributionTestData(
+		addresses,
+		RewardDistributionExpectedValues{
+			MinersSumDiffBalance: MaxAddressReward,
+			DaoDiffBalance:       int64((utl.GetCurrentReward(suite, height) - MaxAddressReward) / 2),
+			XtnDiffBalance:       int64((utl.GetCurrentReward(suite, height) - MaxAddressReward) / 2),
+			Term:                 utl.GetRewardTermAfter20Cfg(suite),
+		})
+}
+
+// 2 miners, dao, initR=700000000, increment = 100000000, desiredR = 900000000
+// ("preactivated_14_19_20/7W_2miners_dao_increase.json")
+// ("preactivated_14_supported_19_20/7W_2miners_dao_increase.json")
+// NODE - 817
+func GetRewardIncreaseDaoTestData(suite *f.BaseSuite, addresses AddressesForDistribution,
+	height uint64) RewardDistributionTestData[RewardDistributionExpectedValues] {
+	return NewRewardDistributionTestData(
+		addresses,
+		RewardDistributionExpectedValues{
+			MinersSumDiffBalance: int64(utl.GetCurrentReward(suite, height)) - MaxAddressReward,
+			DaoDiffBalance:       MaxAddressReward,
+			XtnDiffBalance:       0,
+			Term:                 utl.GetRewardTermAfter20Cfg(suite),
+		})
+}
+
+// 2 miners, xtn, initR=600000000, increment = 100000000, desiredR = 600000000
+// ("preactivated_14_19_20/6W_2miners_xtn_not_changed.json")
+// ("preactivated_14_supported_19_20/6W_2miners_xtn_not_changed.json")
+// NODE - 817
+func GetRewardUnchangedXtnTestData(suite *f.BaseSuite, addresses AddressesForDistribution,
+	height uint64) RewardDistributionTestData[RewardDistributionExpectedValues] {
+	return NewRewardDistributionTestData(
+		addresses,
+		RewardDistributionExpectedValues{
+			MinersSumDiffBalance: int64(utl.GetCurrentReward(suite, height)) - MaxAddressReward,
+			DaoDiffBalance:       0,
+			XtnDiffBalance:       MaxAddressReward,
+			Term:                 utl.GetRewardTermAfter20Cfg(suite),
+		})
+}
+
+// 2 miners, xtn, initR=500000000, increment = 100000000, desiredR = 300000000
+// ("preactivated_14_19_20/5W_2miners_xtn_decrease.json")
+// ("preactivated_14_supported_19_20/5W_2miners_xtn_decrease.json")
+// NODE - 818
+func GetRewardDecreaseXtnTestData(suite *f.BaseSuite, addresses AddressesForDistribution,
+	height uint64) RewardDistributionTestData[RewardDistributionExpectedValues] {
+	return NewRewardDistributionTestData(
+		addresses,
+		RewardDistributionExpectedValues{
+			MinersSumDiffBalance: int64(utl.GetCurrentReward(suite, height)) - int64(utl.GetCurrentReward(suite, height)-MaxAddressReward)/2,
```

Comment:
HTTP request count will be decreased if you create a new variable with result of `utl.GetCurrentReward(suite, height)` function call and reuse it (fix it in all places).

Reply:
Done